extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -198,33 +198,35 @@ export class FileConfigPage extends ImportPage {
 		this.databaseDropdown.values = [];
 		this.schemaDropdown.values = [];
 
-		if (!this.model.server) {
-			//TODO handle error case
-			this.databaseDropdown.loading = false;
-			return false;
-		}
-
-		let defaultServerDatabase = this.model.server.options.database;
-
-		let values: any[];
 		try {
-			values = await this.getDatabaseValues();
-		} catch (error) {
-			// This code is used in case of contained databases when the query will return an error.
-			console.log(error);
-			values = [{ displayName: defaultServerDatabase, name: defaultServerDatabase }];
-			this.databaseDropdown.editable = false;
-		}
+			if (!this.model.server) {
+				//TODO handle error case
+				this.databaseDropdown.loading = false;
+				return false;
+			}
 
-		this.model.database = defaultServerDatabase;
+			let defaultServerDatabase = this.model.server.options.database;
 
-		this.databaseDropdown.updateProperties({
-			values: values
-		});
+			let values: any[];
+			try {
+				values = await this.getDatabaseValues();
+			} catch (error) {
+				// This code is used in case of contained databases when the query will return an error.
```

Comment:
Do you not want to display an error to the user in that case?

Reply:
We are already handling the error by setting the dropdown to the default database of the connection. Do we still need to display this error?